firebaseConfig.js

```diff
@@ -0,0 +1,18 @@
+import { initializeApp, getApps, getApp } from "firebase/app";
+import { getFirestore } from "firebase/firestore";
+
+const firebaseConfig = {
```

Comment:
send this config to me privately and re-generate it. You're not *supposed to place secrets* inside the repository.